spec/models/person_spec.rb

```diff
@@ -202,15 +202,15 @@
       let(:user) { FactoryGirl.create(:person, email: 'user@example.com', phone:'510-555-1234') }
       before { expect(user).to receive(:update_nation_builder).and_call_original }
 
-      it "sends tags to NationBuilder if present" do
+      it "sends remote_fields to NationBuilder if present" do
         expect(NbPersonPushJob).to receive(:perform_later).
-          with("email" => "user@example.com", tags: ['test'])
-        user.update(tags:['test'])
+          with(email: 'user@example.com', tags: ['test'], foo: 'bar')
+        user.update(remote_fields: { tags: ['test'], foo: 'bar' })
       end
 
       it "sends call to update NationBuilder if relevant field changed" do
         expect(NbPersonPushJob).to receive(:perform_later).
-          with("email" => "user@example.com", "first_name"=>"Bob")
+          with(email: 'user@example.com', first_name: 'Bob')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.